src/app/employee/invite-message/invite-message.component.spec.ts

```
@@ -0,0 +1,59 @@
+      providers: [
+        { provide: MatDialogRef, useValue: dialogRefSpy },
+        { provide: MAT_DIALOG_DATA, useValue: { name: 'Test Company', role: 'Admin' } },
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(InvitationDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)